eng/docgeneration/script/CopyReadMes.py

```diff
@@ -0,0 +1,29 @@
+# Copy All ReadMes in the Repo
```

Comment:
Out of curiousity why did you chose python for the scripts? Generally folks working in the .NET repo use powershell.

Reply:
No Particular reason, I also did it in PowerShell initially, but the using python seemed more straightforward.